### PR TITLE
add projectfile, meshes and script

### PR DESCRIPTION
**Pull request by dominik-kern**
The staggered scheme of OGS (currently implemented for the HM-process) comes with a coupling parameter, whose optimal value is a-priori unknown (it depends on sources and boundaries but only negligible on the mesh).
This example demonstrates how to find this optimum iteratively.
It saves time to find the optimal value of the coupling parameter on a coarse mesh and then use it on finer meshes.

### CODE REVIEW

**Comment by dominik-kern** [comment]
@joergbuchwald maybe there is a better way to include the ``search toolbox`` in the ogs6py class.

**Comment by joergbuchwald** [comment]
Requires #32 to be merged first.

File: examples/optimize_coupling_scheme_parameter/search_optimal_coupling_parameter.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Dec 20 13:58:50 2021
+
+@author: dominik
+"""
+from search_toolbox import coupling_parameter_search
+
+ogs_dir = "/home/dominik/OGS/release/bin"
```
Comment (joergbuchwald) [inline examples/optimize_coupling_scheme_parameter/search_optimal_coupling_parameter.py:10]: Should not be user-specific. Either use ogs from search path or use argc/argv.

**Comment by dominik-kern** [comment]
restart

**Pull request closed without merging**